bundles/org.eclipse.emfcloud.modelserver.emf/src/org/eclipse/emfcloud/modelserver/emf/common/DefaultSessionController.java

```diff
@@ -197,39 +225,82 @@ protected void broadcastIncrementalUpdates(final String modeluri, final CCommand
       getOpenSessions(modeluri).forEach(session -> broadcastIncrementalUpdate(session, updates));
    }
 
+   protected void broadcastIncrementalUpdatesV2(final String modeluri, final JsonNode jsonPatch) {
+      // TODO Properly specify the message format. Currently reusing the same signature as V1,
```

Comment:
Pull out Todo into an issue?

Reply:
If possible, this should be handled in the current PR. Let's come back to this once support for Json Patches and Notifications is ready.